starcheck/src/lib/Ska/Parse_CM_File.pm

```diff
@@ -121,20 +123,23 @@ sub dither {
           }
 	$dith_hist_fh->close();
 
-    # If the most recent/last entry in the dither file has a timestamp newer than
-    # the first entry in the load, return string error and undef dither history.
-    if ($dh_time[-1] >= $bs_arr->[0]->{time}){
-       return ("Dither history runs into load\n", undef);
-    }
+    if (not defined $dither_error){
+        # If the most recent/last entry in the dither file has a timestamp newer than
+        # the first entry in the load, return string error and undef dither history.
+        if ($dh_time[-1] >= $bs_arr->[0]->{time}){
+        $dither_error = "Dither history runs into load\n";
+        }
 
-    # Confirm that last state matches kadi continuity ENAB/DISA.
-    # Otherwise, return string error and undef dither history.
-    if ($kadi_dither->{'dither'} ne $dh_state[-1]){
-        return ("Dither status in kadi commands does not match DITHER history\n", undef);
+        # Confirm that last state matches kadi continuity ENAB/DISA.
+        # Otherwise, return string error and undef dither history.
+        if ($kadi_dither->{'dither'} ne $dh_state[-1]){
+            $dither_error = "Dither status in kadi commands does not match DITHER history\n"
```

Comment:
I would take out that newline as it comfortably fits on one line.

Reply:
Is this error shown in red in the HTML report? Otherwise there is a good chance I'd miss it in review. Or I thought there was some section for overall processing errors that this could go in?

Reply:
Oh, and I'll check that newline... I run into the issue that the silly Prev Next links in starcheck will move around if the text block is too wide even if it could fit in a modern browser.